schema/schema.json

```diff
@@ -139,6 +139,31 @@
                 "noNewPrivileges": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
                     "type": "boolean"
+                },
+                "rlimits": {
+                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
+                    "type": "array",
+                    "items": [
+                        {
```

Comment:
And those non-normative docs do have a [list of objects][1], but only for validating fixed-length arrays.  And we want to allow more than one entry in **`rlimits`**.

[1]: http://spacetelescope.github.io/understanding-json-schema/reference/array.html#tuple-validation